### PR TITLE
Patches

### PR DESCRIPTION
**Pull request by OrangeChannel**
Details in commit messages. Fixes some whitespace stuff, some unused (?) parameters, a docstring typo, and version number inconsistency.

### CODE REVIEW

File: lvsfunc/comparison.py [hunk 1]
```diff
@@ -172,12 +170,12 @@ def tvbd_diff(tv: vs.VideoNode, bd: vs.VideoNode,
...
         diff = core.std.PlaneStats(tv, bd)
-        frames = [i for i,f in enumerate(diff.frames()) if f.props["PlaneStatsDiff"] > thr]
+        frames = [i for i, f in enumerate(diff.frames()) if f.props["PlaneStatsDiff"] > thr]
     else:
         diff = core.std.MakeDiff(tv, bd).std.PlaneStats()
-        frames = [i for i,f in enumerate(diff.frames()) if f.props["PlaneStatsMin"] <= thr or f.props["PlaneStatsMax"] >= 255 - thr]
+        frames = [i for i, f in enumerate(diff.frames()) if f.props["PlaneStatsMin"] <= thr or f.props["PlaneStatsMax"] >= 255 - thr]
 
-    if frames == []:
+    if not frames:
```
Comment (LightArrowsEXE) [inline lvsfunc/comparison.py:178]: I'm fairly sure this broke it back when I had it written like this.

File: lvsfunc/misc.py [hunk 2]
```diff
@@ -186,6 +186,7 @@ def limit_dark(clip: vs.VideoNode, filtered: vs.VideoNode,
     :return:                  Conditionally filtered clip
     """
     def _diff(n, f, clip, filtered, threshold, threshold_range):
+        # TODO: Use unused param `n`.
```
Comment (LightArrowsEXE) [inline lvsfunc/misc.py:189]: I think `n` is required to be passed to frame eval, even if it ends up being unused.

File: lvsfunc/scale.py [hunk 3]
```diff
@@ -65,6 +63,7 @@ def _get_error(clip, height, kernel, b, c, taps):
         return descale, diff
 
     def _diff(n, f, clip_a, clip_b, threshold):
+        # TODO: Use unused param `n`.
```
Comment (LightArrowsEXE) [inline lvsfunc/scale.py:66]: See previous comment.

File: lvsfunc/scale.py [hunk 4]
```diff
@@ -147,11 +146,12 @@ def _perform_descale(height: int) -> ScaleAttempt:
     clips_by_resolution = {c.resolution.height: c for c in map(_perform_descale, resolutions)}
     # If we pass a variable res clip as first argument to FrameEval, we’re also allowed to return one.
     variable_res_clip = core.std.Splice([
-        core.std.BlankClip(clip, length=len(clip)-1), core.std.BlankClip(clip, length=1, width=clip.width + 1)
+        core.std.BlankClip(clip, length=len(clip) - 1), core.std.BlankClip(clip, length=1, width=clip.width + 1)
     ], mismatch=True)
 
     def _select_descale(n: int, f: List[vs.VideoFrame]):
-        best_res = max(f, key=lambda frame: math.log(clip.height - frame.props.descaleResolution, 2) * round(1/max(frame.props.PlaneStatsAverage, 1e-12)) ** 0.2)
+        # TODO: Use unused param `n`.
```
Comment (LightArrowsEXE) [inline lvsfunc/scale.py:153]: See previous comment.

File: lvsfunc/scale.py [hunk 5]
```diff
@@ -202,12 +202,13 @@ def smart_reupscale(clip: vs.VideoNode, width: Optional[int] = None, height: int
     c = float(c)
 
     def _transpose_shift(n, f, clip):
+        # TODO: Use unused param `n`.
```
Comment (LightArrowsEXE) [inline lvsfunc/scale.py:205]: See previous comment.

File: lvsfunc/comparison.py (hunk 1 above)
Comment (heicrd) [reply lvsfunc/comparison.py:178]: it should probably just be `if len(frames) == 0`

File: lvsfunc/misc.py (hunk 2 above)
Comment (heicrd) [reply lvsfunc/misc.py:189]: It is and this isn't even a linter warning, so yeah this comment should be removed

File: lvsfunc/comparison.py (hunk 1 above)
Comment (heicrd) [reply lvsfunc/comparison.py:178]: pep8 says to keep it as `if not frames:`

**Pull request merged**